### ISSUE TITLE
Allow filtering without search term

### ISSUE THREAD

**Issue by thetable** [oramasearch/orama#294]
**Is your feature request related to a problem? Please describe.**
In my app, I'd like to support filtering items without applying a search term (e.g. "show me all products between $50 and $100")

**Describe the solution you'd like**
Support a query like this: 
```
{
  "where": {
    "price": {
      "between": [
        50,
        100
      ]
    }
  }
}
```

**Describe alternatives you've considered**
An alternative would be to support the use of wildcard characters in the query language, like this: 
```
{
  "term": "*"
}
```

**Comment by WonderPanda** [comment]
Came here looking for this exact functionality. It would be a huge win for us if we could leverage Lyra for the filtering capabilities even in the absence of a search term.

I feel like having a wild card match using something like `'*'` would be a valid approach and would line up with how property wildcard searching is configured currently

**Comment by thetable** [comment]
There's a practical workaround:

You add an extra `"foo": "bar"` field to every document in the index, then search for `“bar”` in case I only want to filter.

**Comment by micheleriva** [comment]
Available since v1.0.0. Full filters spec: https://docs.oramasearch.com/usage/search/filters
